src/DependencyInjection/CmfSonataAdminIntegrationExtension.php

```diff
@@ -40,14 +40,15 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
 
-        $this->loadBundles($config['bundles'], $loader, $container);
+        $this->loadByBundles($config['bundles'], $loader, $container);
     }
 
-    private function loadBundles(array $config, XmlFileLoader $loader, $container)
+    private function loadByBundles(array $config, XmlFileLoader $loader, ContainerBuilder $container)
```

Comment:
sorry, was a result of a merge conflict, had an almost equal function before i rebased on master with your factories.